docs/installation.asciidoc

```diff
@@ -20,13 +20,48 @@ npm install @elastic/elasticsearch@<major>
 To learn more about the supported major versions, please refer to the 
 <<js-compatibility-matrix>>.
 
+[discrete]
+[[nodejs-support]]
+=== Node.js support
+
+NOTE: The minimum supported version of Node.js is `v8`.
+
+The client versioning follows the Elastic Stack versioning, this means that
+major, minor, and patch releases are done following a precise schedule that
+often does not coincide with the https://nodejs.org/en/about/releases/[Node.js release] times.
+
+For avoiding supporting insecure and unsupported versions of Node.js, the
```

Comment:
```suggestion
To avoid support issues and unsupported versions of Node.js, the
```

Reply:
I want to keep the "insecure", as using an EOL version is not safe :)